netlogo-core/src/test/fileformat/NLogoFormatTests.scala

```diff
@@ -111,6 +111,42 @@ class NLogoFormatConversionTest extends FunSuite with ConversionHelper {
       assert(rereadModel.code.contains("extensions [vid]"))
     }
 
+    test("performs tests for sectionToSource conversion conversion between files") {
+      val sectionsMap = collection.immutable.Map("org.nlogo.modelsection.version" -> Array[String]("NetLogo 3D 6.0.4"))
+      val nlogoformat = new NLogoFormat
```

Comment:
It's confusing to see the `nlogoformat` here, when the `nlogoformatThreed` is declared below.  I think they should be together, probably below the expected results.